src/vizier/services/adaptive_export/cmd/main.go

```
@@ -14,283 +14,473 @@
 	"fmt"
 	"os"
 	"os/signal"
+	"strconv"
+	"strings"
+	"sync"
 	"syscall"
 	"time"
 
 	log "github.com/sirupsen/logrus"
-	"px.dev/pixie/src/api/go/pxapi"
 
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)